functions.php

```diff
@@ -28,6 +28,9 @@ function scripts_and_styles_method() {
     // Enqueue youtube api
     wp_enqueue_script('yt-player-api', 'http://www.youtube.com/player_api', array(), false, true);
 
+    // Enqueue Shuffle.js
+    wp_enqueue_script('shuffle-js', 'https://cdnjs.cloudflare.com/ajax/libs/Shuffle/5.0.2/shuffle.min.js', '', '', true);
```

Comment:
This should be bower installed. only the yt player isn't on bower thats why it is an enqueue

Reply:
did you check to see if shufflejs was one bower or just assume that i hadn’t 🤔

Reply:
yh i checked

Reply:
![screen shot 2017-10-23 at 12 54 02 pm](https://user-images.githubusercontent.com/7727722/31904577-4e395536-b7f1-11e7-90b8-0eed0f4d5034.png)

Reply:
do you want to just ignore that?

Reply:
we can define the dependencies explicitly in `package.json` but idk seems a little sketchy.

Reply:
if thats what you want then commit it